src/Controllers/GagePageController.ts

```diff
@@ -200,7 +204,11 @@ module StreamStats.Controllers {
         public formattedDailyPlusAvg = [];
         public formattedEstPeakDates = [];
         public formattedDailyFlow = [];
+        public dailyRange = [];
+        public formattedDischargePeakDates = []; // Stage vs. Discharge Plot
         public dailyValuesOnly = [];
+        public ageQualityData = 'age'; //Stage vs. Discharge Plot
+        public error: any;
```

Comment:
Could we call this `USGSMeasuredError` or something similar?

Reply:
I would be fine changing the name of the error. But as I search for the word "error" in the code, I see it referenced in other parts that I haven't touched so I feel reluctant to change the name?

Reply:
From what I see the other references to a variable called `error` are all local to the specific functions and not used globally. If you search for `this.error`, which is the global variable `error`, that is just used in the code you added correct? If so, you can remove it as long as you make the other changes I mentioned.

Reply:
Let me know if that doesn't make sense, and I can pull down and push up to show what I mean.

Reply:
Gotcha, I see how the global and local variables work and see what you are asking for! I have tried deleting the global error throughout the code and replacing it with a local variable. Then when I replace` showInLegend: this.error == false`  with `showInLegend: this.measuredObj.length>0`, it goes back to the main problem of `this.measuredObj.length` not having a consistent length and so the radio buttons would pop up when there was no data. 

I think I want to keep the showInLegend based on if the error is there or not. Whenever I replace the global error name with a local error variable, in the showInLegend, it can't connect to the local error variable since the local error is in a totally different function.

Even just changing the global error variable name throughout the code to usgsMeasuredError, the radio buttons won't show up then.. I've also tried other options, but it still doesn't work where the radio buttons show up correctly. I feel that your suggestion of having the radio buttons not show if the error message showed was a great idea. But for that to work, I would need some type of global error variable that I can apply to the statement checking for the error message since it would be looking for that error messsage outside of the function.

I'm just wondering, is there a specific reason why having a local error variable is more valued over a global error variable?

Reply:
I was just hoping we would be able to use the `showInLegend: this.measuredObj.length>0` so it matched with the rest of the plots. Then we just wouldn't have needed the variable to be global since it was only used in the one function. 

If it is not working no worries!